internal/auth/oidc/repository_auth_method_update_test.go

```diff
@@ -129,6 +129,40 @@ func Test_UpdateAuthMethod(t *testing.T) {
 				return am
 			},
 		},
+		{
+			name: "inactive-not-complete-no-with-force",
```

Comment:
Can you add a test which unsets the name and/or description?

Reply:
ty!